<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cloud_to_image_form</class>
 <widget class="QDialog" name="cloud_to_image_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cloud to Image Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="varset_1">
    <item row="0" column="0">
     <widget class="QLabel" name="cloud_in_path_label">
      <property name="text">
       <string>cloud in path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="cloud_in_ledit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="cloud_in_browse_btn">
      <property name="text">
       <string>browse...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="image_out_path_label">
      <property name="text">
       <string>image out path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="image_out_ledit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="image_out_browse_btn">
      <property name="text">
       <string>browse...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>299</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="help_btn">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="launch_test_btn">
      <property name="text">
       <string>Launch test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="image_width_label">
      <property name="text">
       <string>image width</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="image_width_sb">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="image_height_label">
      <property name="text">
       <string>image height</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="image_height_sb">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
